MonoGame.Framework/Graphics/GraphicsDevice.DirectX.cs

```diff
@@ -933,6 +933,18 @@ private void PlatformSetViewport(ref Viewport value)
         }
 
         // Only implemented for DirectX right now, so not in GraphicsDevice.cs
+        public void SetRenderTarget(RenderTarget2D renderTarget, int arraySlice)
+        {
+            if (!GraphicsCapabilities.SupportsTextureArrays)
+                throw new InvalidOperationException("Texture arrays are not supported on this graphics device");
+
+            if (renderTarget == null)
+                SetRenderTarget(null);
+            else
+                SetRenderTargets(new RenderTargetBinding(renderTarget, arraySlice));
+        }
+
+        // Only implemented for DirectX right now, so not in GraphicsDevice.cs
         public void SetRenderTarget(RenderTarget3D renderTarget, int arraySlice)
```

Comment:
This is from [MSDN](https://msdn.microsoft.com/en-us/library/windows/desktop/ff476906%28v=vs.85%29.aspx):

> When a 3D texture mipmap slice is bound as a render target output (with a render-target view), the 3D texture behaves identically to a 2D texture array with n slices. The particular render slice is chosen from the geometry-shader stage, by declaring a scalar component of output data as the SV_RenderTargetArrayIndex system-value.
There is no concept of a 3D texture array; therefore a 3D texture subresource is a single mipmap level.

A `RenderTarget3D`, and a `RenderTarget2D` with `arraySize > 1`, are actually different things, and stored differently. But when you bind them as render targets, they're treated as the same thing.